docs/modules/getting-started/pages/persistence.adoc

```diff
@@ -0,0 +1,192 @@
+= Persist Data on Disk
+:description: This tutorial guides you on enabling persistence in your cluster.
```

Comment:
I would make the first paragraph this description.

Reply:
The description should tell readers why they would want to complete the tutorial.